main.go

```diff
@@ -51,10 +52,6 @@ func main() {
 	}
 
 	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
-	go func() {
```

Comment:
Let's keep this here.

Reply:
After printing `--output-limit-lines` lines, "Received signal, exiting program.." should not be printed.

So while exiting, "Received signal, exiting program.." or "Printed %d events, exiting program.." is printed. They shouldn't be printed both.